pnpm-lock.yaml

```diff
@@ -360,7 +362,7 @@ importers:
       '@prisma/generator-helper': workspace:*
       '@prisma/get-platform': 2.30.0-22.6ee261c643b2c461a20371f48ffa5090ed578aaf
       '@types/jest': 27.0.1
-      '@types/node': 12.20.19
+      '@types/node': 16.6.2
```

Comment:
unrelated? was lockfile just out of sync?

Reply:
Related to undici using `Blob` see previous comment https://github.com/prisma/prisma/pull/8842#issuecomment-903525303

Reply:
Ah, I overlooked the change in `packages/engine-core/package.json`.